ark/analysis/spatial_analysis.py

```diff
@@ -18,20 +21,16 @@ def calculate_channel_spatial_enrichment(dist_matrices_dict, marker_thresholds,
             threshold values for positive marker expression
         all_data (pandas.DataFrame):
             data including fovs, cell labels, and cell expression matrix for all markers
-        excluded_colnames (list):
-            all column names that are not markers. If argument is none, default is
-            ["cell_size", "Background", "HH3",
-            "summed_channel", "label", "area",
-            "eccentricity", "major_axis_length",
-            "minor_axis_length", "perimeter", "fov"]
+        excluded_channels (list):
+            channels to be excluded from the analysis.  Default is None.
         included_fovs (list):
             patient labels to include in analysis. If argument is none, default is all labels used.
         dist_lim (int):
             cell proximity threshold. Default is 100.
         bootstrap_num (int):
             number of permutations for bootstrap. Default is 1000.
         fov_col (str):
-            column with the cell fovs. Default is 'SampleID'
+            column with the cell fovs. Default is equal to `ark.settings.FOV_ID`
```

Comment:
I think we only need to explain the defaults if the value triggers different behavior. For example, if None then takes all images. Otherwise, we're just repeating what's already there.